task/utils.py

```diff
@@ -0,0 +1,13 @@
+import subprocess
+
+def generate_sqlite_hash(sqlite_path):
```

Comment:
Pretty neat really given that we had to call a separate shell process to obtain the sqlite hash